playbooks/roles/jenkins/files/copy_old_builds.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -o errexexit
+
+for d in {release-*,functional-tests-*,visual-regression-*,build-image,database-migration-paas,tag-application-deloyment,clean-and-apply-db-dump-*,index-services-*,index-briefs-*,update-*-index-alias,data-retention-*,database-backup,rotate-api-tokens,rotate-production-notify-callback-token}/; do
```

Comment:
We're missing a couple of jobs from this list (build-scripts, export-data, the smoke tests, a few others) - not sure if this list is meant to be exhaustive or if we're only transferring certain job histories?

Reply:
No, it's only the jobs we really care about the history of. Release pipeline stuff and data backup stuff and a couple of others.